scss/components/table.scss

```diff
@@ -82,20 +85,31 @@ $block: #{$fd-namespace}-table;
             }
         }
     }
-    tbody {
-        tr {
+    &__body {
+        @include fd-reset();
+        .#{$block}__row {
             border-style: solid;
             border-width: $fd-table-border-width;
             @include fd-var-color("border-color", $fd-table-border-color, --fd-table-border-color);
             border-left: none;
             border-right: none;
         }
-        td {
+        .#{$block}__col {
             word-break: break-word;
         }
     }
-    td,
-    th {
+    &__col {
+        line-height: $fd-line-height;
+        font-size: $fd-font-size;
+        -webkit-font-smoothing: antialiased;
+        box-sizing: border-box;
+        padding: 0;
+        margin: 0;
+        border: 0;
+        &::before,
+        &::after {
+            box-sizing: inherit;
+        }
```

Comment:
almost, but not quite - I had to leave out the font color being set in fd-reset and since its only being used once, it didn't seem necessary to add an additional mixin